content/post/career-ladders-part-3.md

```diff
@@ -0,0 +1,57 @@
+- Armed with those benchmarks, we create a growth plan together by agreeing on a few areas where they haven't yet been able to demonstrate the skills or proficiencies for the next level, and they want to focus on improvement. Then we identify concrete projects and actions that will give them an opportunity to build and showcase their skills over the next several months. If they have any ongoing responsibilities or projects that don't align with these growth areas, determine what will happen to those projects - can they be handed to someone who will get more out of them? Can you give them a bigger role on those projects, so that there is still growth opportunity? It takes concerted effort to make space for growth while ensuring all of the team's commitments are met.
+- Keep checking in on those priorities. Keep them in your 1:1 agenda or set monthly reminders to make sure your team member is able to make progress where they want to. Adjust workloads or expectations as necessary. Help your team member solicit feedback from others regularly as well - do the key stakeholders for their work have any interim feedback that will help them improve?
+- About 6 months later, you’ll check back in and see how the plan has gone. Keep a running document, with descriptions or links to work demonstrating various competencies. Hopefully, the assessment looks a little bit different as they’ve worked on a growth area, and you can determine a new area of focus. And if the person is nearing the next level across competencies, you can talk about when that person should be nominated or nominate themselves for a promotion.
+
+These structured conversations help to break down growth plans into achievable scope and align the needs of the organization (in the form of upcoming projects) with the personal desires of your team. They also create a running record of growth that can be used in formal performance reviews and promotion nominations.
+
+### Managing Upward
+Before promotion time approaches, make sure you’re communicating with your manager about where your team members stand and when you’d like to nominate them (or when they’ll likely be nominating themselves, if that’s how your org works). Understand the promotion process, likelihood of promo approval, any limits on the number of promotions (e.g. no more than 20% of employees should be promoted per year), and how you can help your team be ready. Use that information to set expectations appropriately for your team, and find avenues for growth even if it they aren't likely to get a promotion this cycle.
+
+### Promotion Announcements
```

Comment:
Might be worth mentioning the "you're not getting promoted this round" conversation.